torch/csrc/utils/pybind.h

```diff
@@ -325,14 +324,11 @@ struct type_caster<c10::complex<T>> {
   }
 };
 
-// Pybind11 bindings for our optional and variant types.
+// Pybind11 bindings for our optional type.
 // http://pybind11.readthedocs.io/en/stable/advanced/cast/stl.html#c-17-library-containers
 template <typename T>
 struct type_caster<c10::optional<T>> : optional_caster<c10::optional<T>> {};
 
-template <typename... Ts>
```

Comment:
Not sure if it's been removed entirely from the internal codebase yet either so this change may be unsafe.